docs/application/dotnet/guides/nui/quickpanelclient.md

```diff
@@ -0,0 +1,101 @@
+
+## Prerequisites
+To use the methods of the Quickpanel API, use `Tizen.NUI.WindowSystem` in your application:
+```
+using Tizen.NUI.WindowSystem
+```
+
+## Create QuickPanel handle
+After you have created the main window of your application, call `QuickPanelClient()` with a TizenShell and Window to create the handler:
+```
+private void onInitialize()
+{
+    Window window = NUIApplication.GetDefaultWindow();
+    Shell.TizenShell tzShell = new Shell.TizenShell();
+    Shell.QuickPanelClient qpClient = new Shell.QuickPanelClient(tzShell, window, Shell.QuickPanelClient.Types.SystemDefault);
+
+    ... do nice things with qpClient ...
+}
+```
+
+## Show or Hide Quickpanel Window
+To show or hide the Quickpanel service window when your application’s window is activated, use the following code:
+```
+public void qpShowHide()
+{
+    Window window = NUIApplication.GetDefaultWindow();
+    Shell.TizenShell tzShell = new Shell.TizenShell();
+    Shell.QuickPanelClient qpClient = new Shell.QuickPanelClient(tzShell, window, Shell.QuickPanelClient.Types.SystemDefault);
+
+    qpClient.Hide();
+
+    ... doing something without quickpanel ...
+
+    qpClient.Show();
+}
+```
+
+## Get Status of Quickpanel Window
+To get the current status of the Quickpanel service window, you can get status that what you wants like below codes:
+```
+... after qpClient createad ...
+
+VisibleState visible = qpClient.Visible;
+Window.WindowOrientation orientation = qpClient.Orientation;
+ScrollableState scrollable = qpClient.Scrollable;
+```
+Each of status meaning like below.
+* **VisibleState**: the visible state of quickpanel service window. It can be Unknown, Shown, and Hidden
+* **Orientation**: the orientation of quickpanel service window. It has 4 angles, Angle_0, Angle_90, Angle_180, and Angle_270.
+* **ScrollableState**: the scrollable state of quickpanel service window. the detailed scrollable states are like below.
+    * Unknown: Unknown state
+    * Set: The quickpanel service window can scrollable(draggable)
+    * Unset: The quickpanel service window can NOT scrollable
+    * Retatin: The quickpanel service window's scrollable state will be following below visible window's scrollable state.
+
+## Set Scrollable State of Quickpanel Window
+Quickpanel window can be scrollable or not scrollable by Quickpanel Client Window's scrollable state.
+If Scrollable State sets Unset, then the Quickpanel window didn't shown even if the user scrolling(dragging) quickpanel region. To change the scrollable state, use the following code:
+
+```
+... after qpClient createad ...
+
+ScrollableState scrollable = Shell.QuickPanelClient.ScrollableState.Unset;
+```
+
+## Register a Changed Event for Quickpanel Window
+To get notified about the state changes, implement the appropriate event callback function and add to `VisibleChanged`or `OrientationChanged` with that proper event callback function.
```

Comment:
Suggest rephrase, please check if the below suggestion is correct?

To get notified about the state changes, implement the `VisibleChanged`or `OrientationChanged` event callback function.

Reply:
suggested sentence is correct. fixed.